Makefile

```
@@ -13,22 +13,38 @@ GO_FILES := $(shell find . $(DONT_FIND) -o -type f -name '*.go' -print)
 rollout-operator: $(GO_FILES)
 	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=0 go build -ldflags '-extldflags "-static"' ./cmd/rollout-operator
 
+rollout-operator-boringcrypto: $(GO_FILES)
+	GOEXPERIMENT=boringcrypto GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=1 go build -ldflags '-extldflags "-static"' ./cmd/rollout-operator
+
 .PHONY: build-image
 build-image: clean
 	docker buildx build --load --platform linux/amd64 --build-arg revision=$(GIT_REVISION) -t rollout-operator:latest -t rollout-operator:$(IMAGE_TAG) .
 
-.PHONY: publish-image
-publish-image: clean
-	docker buildx build --push --platform linux/amd64,linux/arm64 --build-arg revision=$(GIT_REVISION) -t $(IMAGE_PREFIX)/rollout-operator:$(IMAGE_TAG) .
+.PHONY: build-image
```

Isn't the -boringcrypto suffix missing?
```suggestion
.PHONY: build-image-boringcrypto
```

---

I removed the phony altogether. I don't think we'll have a file called `build-image-boringcrypto` anyways. And IIRC we don't need .phony unless it can collide with a file name and a target